x/capability/keeper/keeper.go

```diff
@@ -67,6 +69,9 @@ func (k *Keeper) ScopeToModule(moduleName string) ScopedKeeper {
 	if k.sealed {
 		panic("cannot scope to module via a sealed capability keeper")
 	}
+	if moduleName == "" {
```

Comment:
is there a reason why `strings.TrimSpace` isn't used?

Reply:
No reason, forgot and added it now. Thanks for the catch